src/private/WriteOutput.ps1

```diff
@@ -0,0 +1,19 @@
+function WriteOutput {
+    [CmdLetBinding()]
+    Param (
+        [Parameter(Position=0,ValueFromPipeline=$true,ValueFromPipelineByPropertyName=$true)]
+        [object]$Output,
+        [Parameter(Position=1,ValueFromPipelineByPropertyName=$true)]
+        [string]$OutputType = "default"
```

Comment:
I don't love that this is a string. This really should be an enum or validated set.